### PR TITLE
MN-581: create method deposit.transferToDeposit

### PR DESCRIPTION
**Pull request by pavlushev**
what I did:
- create new internal method deposit.transferToDeposit

how to verify it:
- launch functests

### CODE REVIEW

File: application/builtin/contract/member/member.go [hunk 1]
```diff
@@ -301,6 +304,46 @@ func (m *Member) depositTransferCall(params map[string]interface{}) (interface{}
...
+	if !ok {
+		return nil, fmt.Errorf("failed to get 'fromDepositName' param")
+	}
+	toDepositName, ok := params["toDepositName"].(string)
+	if !ok {
+		return nil, fmt.Errorf("failed to get 'toDepositName' param")
+	}
+	toMember, ok := params["toMemberReference"].(string)
+	if !ok {
+		return nil, fmt.Errorf("failed to get 'toDepositName' param")
```
Comment (megge-dream) [inline application/builtin/contract/member/member.go:318]:
```suggestion
		return nil, fmt.Errorf("failed to get 'toMemberReference' param")
```

Comment (pavlushev) [reply application/builtin/contract/member/member.go]: yep

File: vendor/modules.txt [hunk 2]
```diff
@@ -104,10 +104,18 @@ github.com/insolar/go-actors/actor/system
 # github.com/insolar/insconfig v0.0.0-20200227134411-011eca6dc866
 github.com/insolar/insconfig
 # github.com/insolar/insolar v1.6.6-0.20200727081822-68505e6410cb
+<<<<<<< HEAD
```
Comment (LinnikD) [inline vendor/modules.txt:107]: conflict

Comment (pavlushev) [reply vendor/modules.txt]: soryamba

**Pull request merged**